### PR TITLE
Adding first PRs visualization

### LINKED ISSUE #15: Visualize Number of PRs opened by new contributors
| Index | Data | Possible ways to obtain it | Visualization | 
|---|---|---|---|
| 3 | Number of PRs oppened by new contributors | Discover if it's the first PR of the user |
| 4 | How many of these first PRs were accepted | Just see if it was accepted |  [Donut Chart](https://datavizcatalogue.com/methods/donut_chart.html)

A donut chart showing how many PRs were opened by new contributors (the first PR by the user). Each section of the donut will show how many were accepted, rejected, or are open (waiting for review).

_Originally posted by @hpbl in https://github.com/hpbl/OnboardingVisualization/issues/4#issuecomment-439728712_

### PR DESCRIPTION
**Pull request by rrbarioni**
closes #15

### CODE REVIEW

File: src/15_visualize_number_prs_new_contributors.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+async function getPrs(user, repo) {
+  let prsList = [];
+  let page = 1;
+  let promiseValue = [0];
+  while (promiseValue.length !== 0) {
+    const fetchResult = await fetch(`https://api.github.com/repos/${user}/${repo}/pulls?access_token=${token}&page=${page}&per_page=100&state=all&sort=created_at`);
+    const promise = fetchResult.json();
+    promiseValue = await promise.then(value => value);
+
+    console.log(`page: ${page}, ${promiseValue.length}`);
```
Comment (hpbl) [inline src/15_visualize_number_prs_new_contributors.js]: please remove console.log statements

File: src/15_visualize_number_prs_new_contributors.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    if (pr.merged_at !== null) {
+      acceptedFirstPrs = acceptedFirstPrs.concat(pr);
+    }
+  }
+  const acceptedPrsNewContributorsCount = acceptedFirstPrs.length;
+
+  const notAcceptedPrs = prsNewContributorsCount - acceptedPrsNewContributorsCount;
+  const acceptedPrs = acceptedPrsNewContributorsCount;
+
+  // console.log(`notAcceptedPrs: ${notAcceptedPrs}`);
```
Comment (MVictorM) [inline src/15_visualize_number_prs_new_contributors.js]: remove comments

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,21 @@
 // Use a Node.js core library
 import * as d3 from 'd3';
+import 'babel-polyfill';
+
 import changeH1Color from './teste';
+import numberPrsNewContributorsAccepted from './15_visualize_number_prs_new_contributors';
```
Comment (hpbl) [inline src/index.js]: I think we can find a better name for the file

File: src/index.js [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
 d3.select('body').style('background-color', 'lightblue');
 
 console.log(d3.version);
 console.log('testando');
 
-
 changeH1Color();
+
+numberPrsNewContributorsAccepted(
+  'CMU-Perceptual-Computing-Lab',
```
Comment (MVictorM) [inline src/index.js]: create global variables for project name, username and use them here. Other functions may use them too

**Comment by MVictorM** [comment]
teste

**Pull request merged**